src/raylib.h

```diff
@@ -1149,6 +1149,7 @@ RLAPI int GetTouchX(void);                                    // Get touch posit
 RLAPI int GetTouchY(void);                                    // Get touch position Y for touch point 0 (relative to screen size)
 RLAPI Vector2 GetTouchPosition(int index);                    // Get touch position XY for a touch point index (relative to screen size)
 RLAPI int GetTouchPointId(int index);                         // Get touch point identifier for given index
+RLAPI bool GetTouchPointState(int index);                     // Patch for touch events in web browsers
```

Comment:
This addition does not seem to belong to this PR.

Reply:
Same. Changes I made afterwards not meant to be part of this PR. Actually, this is in relation to this issue: https://github.com/raysan5/raylib/issues/3100